plugins-compat-tester-cli/src/main/java/org/jenkins/tools/test/CliOptions.java

```diff
@@ -214,4 +220,17 @@ public File getTestJDKHome() {
     public String getTestJavaArgs() {
         return testJavaArgs;
     }
+
+    public List<Plugin> getOverridePlugins() {
+        return overridePlugins;
+    }
+
+    public static class PluginConverter implements IStringConverter<Plugin> {
+        @Override
+        public Plugin convert(String s) {
+            String[] details = s.split("=");
+            assert details.length == 2;
```

Comment:
Asserts will be disabled in the most of JVM defaults. It's better to formally throw an exception